dev/functions.php

```diff
@@ -101,44 +101,53 @@ function wprig_setup() {
 	 *
 	 * @link https://wordpress.org/gutenberg/handbook/extensibility/theme-support/
 	 */
-	add_theme_support( 'editor-color-palette',
+	add_theme_support( 'editor-color-palette', array(
 		array(
-			'name'  => 'Dusty orange',
-			'color' => '#ED8F5B',
+			'name'  => esc_html__( 'Dusty orange', 'wprig' ),
```

Comment:
Using `esc_html__()` is unnecessary here (and in the other similar occurrences), escaping should always happen right before outputting which is not what we're doing here. Simply using `__()` is more appropriate.

Reply:
@felixarntz Done ;)